llvm/docs/CommandGuide/lit.rst

```diff
@@ -63,17 +63,17 @@ GENERAL OPTIONS
 
  Show :program:`lit`'s version number and exit.
 
-.. option:: -j N, --workers=N
+.. option:: -j N, --workers N
 
  Run ``N`` tests in parallel.  By default, this is automatically chosen to
  match the number of detected available CPUs.
 
-.. option:: --config-prefix=NAME
+.. option:: --config-prefix NAME
 
  Search for :file:`{NAME}.cfg` and :file:`{NAME}.site.cfg` when searching for
  test suites, instead of :file:`lit.cfg` and :file:`lit.site.cfg`.
 
-.. option:: -D NAME[=VALUE], --param NAME[=VALUE]
+.. option:: -D NAME [VALUE], --param NAME [VALUE]
```

Comment:
Is this change correct? I think `=` here is part of the parameter argument.

Reply:
I am a lousy programmer it seems. But luckily you're a great reviewer. You're absolutely right. This HAS to stay.

Reply:
This is fixed now.